lib/web_ui/lib/src/engine/canvaskit/text.dart

```diff
@@ -797,6 +797,23 @@ class CkParagraph extends SkiaObject<SkParagraph> implements ui.Paragraph {
     }
     return result;
   }
+
+  bool _disposed = false;
+
+  @override
+  void dispose() {
+    delete();
+    didDelete();
+    _disposed = true;
+  }
+
+  @override
+  bool get debugDisposed {
+    if (assertionsEnabled) {
```

Comment:
For my own curiosity, is assertionsEnabled something internal to this part of the engine?

Reply:
it's defined in `util.dart` for the web engine.

Reply:
TIL thank you!